webpack.config.minified.js

```diff
@@ -0,0 +1,15 @@
+var webpack = require('webpack');
+var _ = require('lodash');
+
+module.exports = _.merge(require('./webpack.config.js'), {
+  output: {
+    filename: 'radium.min.js'
+  },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin({
```

Comment:
Also suggest:

```js
new webpack.optimize.DedupePlugin()
```

now and in the future, consider writing "dev-only" code against a `production.env.NODE_ENV` check and then defining this:

```js
    new webpack.DefinePlugin({
      "process.env": {
        // Signal production mode for React JS and other libs.
        NODE_ENV: JSON.stringify("production")
    })
```

As that's more-or-less the react standard now...